 {% include navigation.html %}
 

## Large-Scale GxE Methods Project
[Link to our project on NIH Reporter](https://reporter.nih.gov/search/gAoJB8GZC023oCHDlwkpCQ/project-details/10199014)

Welcome to the website for the Large-scale GxE Methods Project

## Research Question
The major goals of this project are to develop efficient methods and algorithms for large-scale gene-environment interaction studies, and implement them in open-source software programs and cloud-based analysis pipelines, to facilitate gene-environment interaction research on complex cardio-metabolic, lung, blood and sleep diseases and related conditions using hundreds of thousands to millions of samples.  

## Team members and collaborators 
Han Chen, Alanna Morrison, Yan Sun, and Yun-Ju Sung

## Approach 
Gene- environment interaction (GEI) studies allow us to understand the relationship between genetic variations and a phenotype in the presence of an exposure, which can be a lifestyle (e.g. cigarette smoking), an environmental exposure (e.g. toxin), a physiological factor (e.g. obesity), or a treatment intervention (e.g. daily aspirin therapy).

We plan to develop and implement computationally efficient statistical methods and user-friendly software packages to solve analytical challenges.

## Results
We developed a new software program, GEM (Gene-Environment interaction analysis in Millions of samples) as part of R01 HL145025 (Multi-PIâ€™s Manning/Chen), which supports the inclusion of multiple GEI terms and adjustment for GEI covariates, conducts both model-based and robust inference procedures, and enables multi-threading to reduce computational time.

Through simulations, we demonstrate that GEM enables genome-wide GEI analysis that scales to millions of samples while addressing limitations of existing software programs. In order to further assess the value of genome-wide GEI testing, we conducted a gene-by-sex interaction analysis on body mass index (BMI)-adjusted waist-hip ratio (WHR) in 352,768 unrelated individuals of European ancestry from the UK Biobank.

By testing for sex-by-genotype interaction, we identified 6 novel loci that have not been previously reported as sex-dimorphic for WHR. Using the joint test of both genetic and GEI effects, we identified 39 novel loci that have not been previously reported in sex-specific or combined analyses. Furthermore, through analysis of down-sampled datasets, we showed that the degree of polygenicity is similar for interaction and marginal effects. Our results demonstrate the value of explicit GEI testing in large sample sizes and shed light on the polygenic architecture of sex interaction effects for WHR.
